maui-toc.html

```diff
@@ -1255,6 +1255,8 @@
             <li><a href="/maui/TreeMap/selection-and-highlights">Selection and Highlight</a></li>
             <li><a href="/maui/TreeMap/events">Events</a></li>
```

Comment:
Change **Drilldown** order next to **Levels**
Change **RTL** order before to **Events**

Reply:
Updated changes